openreview/conference/webfield.py

```diff
@@ -88,6 +88,37 @@ def set_home_page(self, group, layout, options = {}):
             group.signatures = [group.id]
             return self.client.post_group(group)
 
+    def set_expertise_selection_page(self, conference, invitation):
+
+        default_header = {
+            'title': conference.get_short_name() + ' Expertise Selection',
+            'instructions': '<p class="dark">Listed below are the papers we have for you. <b>By default, we consider \
+                all of these papers to formulate your expertise.\
+                <br>Please click on \"Exclude\" for papers that should NOT be used to represent your expertise.</b><br>\
```

Comment:
What is the `</b>` tag for?

Reply:
It was a stray bold tag. Thanks for noticing that.